.travis.yml

```diff
@@ -6,8 +6,10 @@ services:
 jobs:
  include:
   - stage: Base
```

Comment:
Now that we have multiple Verificarlo images, might be worth renaming this stage `Verificarlo` and the build names as the OS version.